src/logger/src/metrics.rs

```diff
@@ -87,18 +87,16 @@ pub struct Metrics<T: Serialize> {
     // Metrics will get flushed here.
     metrics_buf: Mutex<Option<Box<dyn Write + Send>>>,
     is_initialized: AtomicBool,
-    pub app_metrics: T,
+    pub metrics_buf_input: T,
```

Comment:
Mhh, the only reason this name makes sense is because of the todo-comment, which this PR removes. Maybe `metrics_data` makes more sense?